packages/experiments/src/components/Stack/examples/Stack.Vertical.Grow.Example.tsx

```diff
@@ -1,35 +1,40 @@
 // @codepen
 import * as React from 'react';
 import { Stack } from '../Stack';
-import { mergeStyleSets, DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
+import { IStackSlots } from '../Stack.types';
+import { IStackItemSlots } from '../StackItem/StackItem.types';
+import { IComponentStyles } from '../../../Foundation';
+import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
 
 export class VerticalStackGrowExample extends React.Component<{}, {}> {
   public render(): JSX.Element {
-    const styles = mergeStyleSets({
+    const rootStyles: IComponentStyles<IStackSlots> = {
       root: {
         background: DefaultPalette.themeTertiary,
         height: 250
-      },
+      }
+    };
 
-      item: {
+    const itemStyles: IComponentStyles<IStackItemSlots> = {
+      root: {
         display: 'flex',
         alignItems: 'center',
         justifyContent: 'center',
         background: DefaultPalette.themePrimary,
         color: DefaultPalette.white
       }
-    });
+    };
 
     return (
       <Stack gap={5}>
-        <Stack gap={5} padding={10} className={styles.root}>
-          <Stack.Item grow={3} className={styles.item}>
+        <Stack gap={5} padding={10} styles={rootStyles}>
```

Comment:
I'd really recommend we avoid suggesting using styles like this for Stack examples. It deemphasizes the important parts of stack.